app/views/rapidfire/questions/_form.html.erb

```diff
@@ -15,6 +15,9 @@
   <%= f.label :question_text %>
   <%= f.text_field :question_text %>
 
+  <%= f.label :position %>
+  <%= f.text_field :position %>
```

Comment:
@leonhooijer can use `number_field` instead of `text_field` as we are expecting integer in db.